src/renderer/index.tsx

```diff
@@ -1,91 +1,59 @@
-import React, { useState, useEffect, useReducer } from 'react';
-import { connect } from 'react-redux';
+import React, { useState, useEffect } from 'react';
 import { Overview } from './overview';
 import { FileListView } from './fileListView/fileListView';
 import { IPC_EVENTS } from '../constants';
 import { ipcRenderer } from 'electron';
-import { SiteImageData, OptimizerStatus } from '../types';
-import { Preferences } from '../types';
-import { scanImageReducer, initialState, SCAN_IMAGES_ACTIONS } from './reducers/scanImageReducer';
-import * as LocalRenderer from '@getflywheel/local/renderer';
-import { fileListReducer } from './reducers/fileListReducer';
-import { POPULATE_FILE_LIST } from './reducers/fileListReducer';
-import { DatasetType } from '../types';
+import { OptimizerStatus, ImageData } from '../types';
+import { store, actions, selectors, useStoreSelector } from './store';
```

Comment:
Hot thought: I think that finding a nice elegant way to bind these to `store` and `dispatch `would be so fantastic for scaling. I know I'm kind of hung up on that, but that is a nice abstraction since not passing `store` or `dispatch`ing is an easy pitfall/bug cause.

Reply:
Maybe Redux Toolkit does it differently, but i thought using redux's `connect(...)` is where binding actions and selectors were setup for a component.